setup.cfg

```diff
@@ -70,6 +70,7 @@ speedups =
   aiodns >= 3.2.0; sys_platform=="linux" or sys_platform=="darwin"
   Brotli; platform_python_implementation == 'CPython'
   brotlicffi; platform_python_implementation != 'CPython'
+  hyperscan >= 0.7.8; platform_python_implementation == 'CPython' and (sys_platform=="linux" or sys_platform=="darwin") and (python_version < "3.14")
```

Comment:
Do we want the <3.14 here? Seems like something that'll be easy to forget.

Reply:
hyperscan doesn't exist for 3.14 yet. 
I'll try to make tests passed on 3.14. 
Now they fails on pydantic installation; it presents in our requirement files but never used by code. I guess the dependency could be removed in a separate PR.

Reply:
3.14 test are unlikely to pass until we are close to release. I just put it in early in preparation. But, we'd need wheels for multidict/yarl etc. which will only happen when 3.14rc is available.

Reply:
OK, I see your point.

Reply:
Done